collect_app/src/main/java/org/odk/collect/android/projects/ProjectImporter.kt

```diff
@@ -49,15 +55,11 @@ class ProjectImporter(
         settingsProvider.getGeneralSettings(project.uuid).saveAll(generalSharedPrefs.all)
         settingsProvider.getAdminSettings(project.uuid).saveAll(adminSharedPrefs.all)
 
-        setupProject(project)
-        return project
-    }
-
-    fun setupProject(project: Saved) {
         createProjectDirs(project)
+        return project
     }
 
-    private fun createProjectDirs(project: Saved) {
+    fun createProjectDirs(project: Saved) {
```

Comment:
I think this should be private right? It looks like a test helper calls it, but it could just use `importNewProject` instead I think.

Reply:
Fixed.